core/src/constants/COLLATERALS.ts

```diff
@@ -201,6 +201,26 @@ const COLLATERALS: Record<string, CollateralConfig> = {
             route: ['ETH'],
         },
     },
+    'WBTC-B': {
```

Comment:
Do we have any kind of specific order in which we add these collaterals? I saw you moved these two collaterals further up.

Reply:
We don't have a specific order, but it's the same order collaterals are shown in the dashboard, so I just moved `WBTC-*` collaterals together